region/lowernorfair/east/Lower Norfair Spring Ball Maze Room.json

```
@@ -437,7 +428,9 @@
       "id": 17,
       "link": [3, 3],
       "name": "Open Door",
-      "requires": [],
+      "requires": [
+        {"heatFrames": 0}
```

```suggestion
        {"heatFrames": 20}
```
[3, 7] that opens the door first would add two turnarounds.  It may mess with your alcoon dodge strat too so its something to look at for the room later.